middleware/http/source/outbound/transform.cpp

```diff
@@ -18,13 +18,14 @@ namespace casual
          {
             auto header( const std::vector< configuration::Header>& model)
             {
-               common::service::header::Fields headers;
-
-               common::algorithm::transform( model, headers.container(), []( auto& h){
-                  return common::service::header::Field{ h.name, h.value};
-               });
-
-               return std::make_shared< const common::service::header::Fields>( std::move( headers));
+               auto transform_fields = []( auto model)
```

Comment:
Very minor, but how come you aren't taking `model` by reference when the surrounding function does?

Reply:
Must be a mistake.